src/data/faq_de.json

```diff
@@ -14,6 +14,79 @@
+                            "Die Corona-Warn-App nutzt Funktionen der Betriebssysteme von Google und Apple, um die Risikoberechnung durchzuführen. Diese Funktionen bieten aus Datenschutzgründen keine Möglichkeit Orts- und Uhrzeitangaben aufzuzeichnen.<br>Sie erhalten die Information, an welchem Tag letztmalig eine Risikobegegnung stattgefunden hat. Orte und Personen können Sie im Kontakttagebuch eintragen. Weitere Informationen erhalten Sie hier: <a href ='#personal_data' target='_blank'>Welche personenbezogenen Daten werden in der App gespeichert?</a>",
+                            "<b>Wie häufig wird der Risikostatus aktualisiert?</b>",
+                            "Der Risikostatus wird bis zu 6 mal täglich automatisch aktualisiert, wenn eine WLAN-Verbindung vorhanden ist. Bei einer mobilen Internetverbindung wird der Risikostatus lediglich einmal täglich abgefragt. Weitere Informationen erhalten Sie hier: <a href='#risk_update_24' target='_blank'>Wie oft wird der Risiko-Status aktualisiert?</a>",
+                            "<b>Wie entstehen unterschiedliche Risikobewertungen bei Nutzern, die sich am gleichen Ort aufgehalten haben?</b>",
+                            "Abweichende Infektionsrisikobewertungen können mehrere Ursachen haben, beeinträchtigen aber nicht die Wirksamkeit der Corona-Warn-App.<br>Die Einschätzung eines möglichen Infektionsrisikos hängt von der erkannten Kontaktdauer und dem ermittelten Abstand zwischen den Smartphones ab. Die Ermittlung des Abstandes wird dabei auch von der Ausrichtung der beiden Smartphone-Antennen zueinander beeinflusst. Daher ist dieser Aspekt der Risikobewertung sehr stark von der exakten räumlichen Lage der beteiligten Smartphones abhängig. Weitere Informationen erhalten Sie hier: <a href='#dissimilar_indication_of_risk_status' target='_blank'>Unterschiedliche Anzeige von Risiken</a>",
+                            "<br>",
+                            "<h3>Testen</h3>",
+                            "<hr>",
+                            "<b>Habe ich Anspruch auf einen PCR-Test?</b>",
+                            "Nach Rücksprache mit Ihrer Hausarztpraxis, dem kassenärztlichen Bereitschaftsdienst (Telefonnummer 116 117), oder Ihrem Gesundheitsamt kann gegebenenfalls ein Test veranlasst werden. Es handelt sich dabei nicht notwendigerweise um einen PCR-Test.  ",
+                            "<b>Wo kann ich mich testen lassen?</b>",
+                            "An die Corona-Warn-App angebundene Teststellen können Sie hier finden: <a href='map.schnelltestportal.de' target='_blank'>map.schnelltestportal.de</a><br>Eine weitere Übersicht von Teststellen finden <a href='https://www.zusammengegencorona.de/testen/?' target='_blank'>Sie auf dieser Seite des Bundesministeriums für Gesundheit</a>. Bei den dort aufgeführten Teststellen ist eine Anbindung an die Corona-Warn-App jedoch nicht garantiert.",

```

Comment:
Take "Sie auf" out of hyperlink

`<a href='https://www.zusammengegencorona.de/testen/?' target='_blank'>Sie auf dieser Seite des Bundesministeriums für Gesundheit</a>`

Reply:
Thanks for your suggestion. In the latest commit, the entire line has been edited so that there is just one link left that links to the matching FAQ article.